notebook/notebook/notebook.py

```diff
@@ -66,6 +76,36 @@ def get_name(line):
         "working directory must contain a file called `notebook-worker-images' "
         "containing the name of the docker image to use for worker pods.") from e
 
+CREATED_STATE = 'created'
+DELETED_STATE = 'deleted'
+
+AUTH_GATEWAY = os.environ.get("AUTH_GATEWAY")
+DB = os.environ.get("MYSQL_DB")
+IP = os.environ.get("MYSQL_IP")
+PORT = int(os.environ.get("MYSQL_PORT"))
+USER = os.environ.get("MYSQL_USER")
+PASS = os.environ.get("MYSQL_PASS")
+TABLE = 'sessions'
+mysql_conn = pymysql.connect(host=IP, port=PORT, user=USER, passwd=PASS, db=DB)
+
+cur = mysql_conn.cursor()
+
+cur.execute("""
+    CREATE TABLE IF NOT EXISTS sessions (
+        user_id VARCHAR(255),
+        pod_name VARCHAR(255),
+        svc_name VARCHAR(255),
+        token VARCHAR(255),
+        state VARCHAR(255),
+        name VARCHAR(255),
+        PRIMARY KEY (user_id, token)
```

Comment:
Seems odd that the primary key isn't pod_name. That's the uniquely identifying piece of information.